### PR TITLE
UserDataService compleated

### PR DESCRIPTION
**Pull request by Fayzulloabdullayev09**
UserDataService compleated

### CODE REVIEW

File: TheConversationsBot/SQLQuery/SQLQuery.sql
Comment (Zuxriddin-Zarifov) [inline TheConversationsBot/SQLQuery/SQLQuery.sql:1]: bular nima?

Comment (Fayzulloabdullayev09) [reply TheConversationsBot/SQLQuery/SQLQuery.sql:1]: Nmani aytyapsz

Comment (Zuxriddin-Zarifov) [reply TheConversationsBot/SQLQuery/SQLQuery.sql:1]: 7 changet bolganmi?

Comment (Zuxriddin-Zarifov) [reply TheConversationsBot/SQLQuery/SQLQuery.sql:1]: shularni hammasini siz yozdizmi shunda ☺

**Pull request closed without merging**